Automattic/jetpack
# JITM: wrap CTA below text on small viewports

Fixes 1164141197617539-as-1201305007019335

#### Changes proposed in this Pull Request:
This PR fixes the layout of JITM banners on screens narrower than 480px, by wrapping the CTA under the text.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Start with a site that's connected to WordPress.com.
* Go to a wp-admin page with a JITM message, like the main wp-admin page or Jetpack > Dashboard.
* Check how the message looks like on desktop and on mobile devices.

_Before_
![image](https://user-images.githubusercontent.com/426388/143571066-cc93fa08-b7b4-4c29-9174-da254e3197b2.png)

_After_
![image](https://user-images.githubusercontent.com/426388/143570783-5a3ca3b4-7856-402e-93a2-0a72636612e3.png)
